### PR TITLE
FEATURE: Cleanup docs

### PR DESCRIPTION
**Pull request by DanielSiepmann**
We didn't cover docs for all changes and therefore update them now.

### CODE REVIEW

File: Documentation/source/development/contribution.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* MySQL is up and running with user *dev* and password *dev* on *127.0.0.1* or to overwrite the
+  environment variables, see :file:`Makefile`.
+  And MySQL is not set to strict mode as TYPO3 doesn't support strict mode, see
+  https://review.typo3.org/#/c/26725/3/INSTALL.md.
+
+* Elasticsearch is installed and up and running on *localhost:9200*.
+
+Then setup your system::
+
+    git clone git@github.com:DanielSiepmann/search_core.git \
```
Comment (justusmoroni) [inline Documentation/source/development/contribution.rst]: Falsch? `DanielSiepmann/search_core`?

File: Documentation/source/development/contribution.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+* New tests whenever possible and useful.
+
+* Code has to follow `PSR-2`_.
+
+* Adjusted documentation.
+
+* Make sure to follow the documented :ref:`concepts`.
+
+.. _Github: https://github.com/DanielSiepmann/search_core
```
Comment (justusmoroni) [inline Documentation/source/development/contribution.rst]: Auch hier. `DanielSiepmann`?

File: Documentation/source/development/indexer.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Each indexer has to be a single class which implements
+``Codappix\SearchCore\Domain\Index\IndexerInterface``.
+
+The indexer should call the connection with all necessary information about the document(s) to
+trigger indexing or deletion of whole index.
+
+As this is the "indexer", deletion of single documents is directly processed by the connection.
+
+``setIdentifier`` is called with the identifier of the current Indexer. This might be usefull to
```
Comment (justusmoroni) [inline Documentation/source/development/indexer.rst]: `useful` ohne 2 `f`.

File: Documentation/source/readme.rst [hunk 4]
```diff
@@ -8,24 +8,21 @@ What does it do?
...
 
 Current state
 -------------
 
 This is still a very early beta version. More information can be taken from Github at
-`current issues`_ and `current projects`_.
+`current issues`_.
 
-We are also focusing on Code Quality and Testing through `travis ci`_, `scrutinizer`_ and `codacy`_.
+We are also focusing on Code Quality and Testing through `travis ci`_, ``phpcs``, ``phpunit`` and
```
Comment (justusmoroni) [inline Documentation/source/readme.rst:24]: Hier fehlt nun `phan` oder?

**Review by justusmoroni — CHANGES_REQUESTED** [review]

File: Documentation/source/readme.rst (hunk 4 above)
Comment (DanielSiepmann) [reply Documentation/source/readme.rst:24]: phan does not work completely yet. There are some issues regarding TYPO3, so therefore it's not mentioned yet, still configuration is already there to use on local machine.

File: Documentation/source/development/contribution.rst (hunk 1 above)
Comment (DanielSiepmann) [reply Documentation/source/development/contribution.rst]: 👍

File: Documentation/source/development/contribution.rst (hunk 2 above)
Comment (DanielSiepmann) [reply Documentation/source/development/contribution.rst]: 👍

File: Documentation/source/development/indexer.rst (hunk 3 above)
Comment (DanielSiepmann) [reply Documentation/source/development/indexer.rst]: 👍

**Review by justusmoroni — APPROVED** [review]

**Pull request merged**